permissions/plugin-aws-beanstalk-releaser.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "aws-beanstalk-releaser"
+github: "jenkinsci/aws-beanstalk-releaser-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-beanstalk-releaser"
```

Comment:
This is incorrect, your path is the `groupId`/`artifactId` (with periods replaced with slashes) from your pom. In your pom (https://github.com/jenkinsci/aws-beanstalk-releaser-plugin/blob/master/pom.xml#L10) the `groupId` is `io.jenkins.plugins` so the path would be io/jenkins/plugins/aws-beanstalk-releaser`

Reply:
I fixed the pom.xml since it was just a pregenerated value.
Now you should find 'org.jenkins-ci.plugins' as groupid in the pom.

Reply:
io.jenkins.plugins is the preferred groupId unless you want to use one specific to your company. If you want to use the generic Jenkins groupId, we are migrating to the io.jenkins.plugins one going forward. Please either update to a custom groupId if your company has one, or use io.jenkins.plugins.

Reply:
Ok I reverted the groupId and now path is io/jenkins/plugins/aws-beanstalk-releaser